src/content/docs/iast/install.mdx

```diff
@@ -15,85 +15,115 @@ freshnessValidatedDate: never
-      >
-        ```yml
-        security:
-            enabled: true
-            agent:
-                enabled: true
         ```
-      </Collapser>
-
-      <Collapser
-        id="config-eu-file-example"
-        title={<><InlineCode>newrelic.yml</InlineCode> config file for EU</>}
-      >
-        ```yml
         security:
-            enabled: true
-            agent:
-                enabled: true
-            validator_service_url: wss://csec.eu01.nr-data.net
+        enabled: true
+        agent:
+            enabled: true 
         ```
-      </Collapser>
-
-      <Collapser
-        id="config-file-fed-example"
-        title={<><InlineCode>newrelic.yml</InlineCode> config file for Fed users</>}
-      >
-        ```yml
-        security:
-            enabled: true
-            agent:
-                enabled: true
-            validator_service_url: wss://csec-gov.nr-data.net
+            </TabsPageItem>
+            <TabsPageItem id="2">
+            Set the `security.enabled` and `security.agent.enabled` flag to true as follows in the `newrelic.js` config file present at the root directory of your application.
+
+            ```
+        security: {
+        agent: {
+            enabled: true,
+        },
+        enabled: true,}
+            ```
+            
+            
+            </TabsPageItem>
+            <TabsPageItem id="3">
+
+        1. Add this integration to your application by importing the following package.
+
+            ```
+            import "github.com/newrelic/go-agent/v3/integrations/nrsecurityagent"
+            ```
+        2. Enable IAST testing by adding code to initialize the integration after your call to `newrelic.NewApplication`.
+
+            ```
+            app, err := newrelic.NewApplication(newrelic.ConfigAppName("Your Application Name"),newrelic.ConfigLicense("NEW_RELIC_LICENSE_KEY"),)
         ```
-      </Collapser>
-    </CollapserGroup>
-  </Step>
 
+        3. Initialize the `nrsecurityagent` as given below:  
+            ```
+            err := nrsecurityagent.InitSecurityAgent(
+            app,
+            nrsecurityagent.ConfigSecurityEnable(true),
+            nrsecurityagent.ConfigSecurityValidatorServiceEndPointUrl("wss://csec.nr-data.net"),
+            )
+            ```
+
+        4. Ensure that you wrap your framework router with the WrapRouter function for [Echo](https://pkg.go.dev/github.com/newrelic/go-agent/v3/integrations/nrecho-v4#WrapRouter), [Gin](https://pkg.go.dev/github.com/newrelic/go-agent/v3/integrations/nrgin#WrapRouter), [Gorillaframeworks](https://pkg.go.dev/github.com/newrelic/go-agent/v3/integrations/nrgorilla#WrapRouter). 
+        This is crucial for detecting routes(API endpoints) and enabling provable security in your application.
+        5. If you are opening an HTTP protocol endpoint, place the  newrelic.WrapListen function around the endpoint name to enable vulnerability scanning against that endpoint. For example:
```

Comment:
In general, we prefer contractions, like "you're".

Also, I think `newrelic.WrapListen` should be surrounded by backticks